src/Fetcher/ConfTechFetcher.php

```diff
@@ -0,0 +1,209 @@
+
+        foreach ($fetchedConferences as $fC) {
+            $fC = $this->hash($fC);
+            $fC->tag = $tag;
+            $fC->source = $source;
+
+            $conference = $this->hydrateConference($fC);
+
+            $conferences[$conference->getHash()] = $conference;
+        }
+
+        return $conferences;
+    }
+
+    private function matchTags()
+    {
+        $dateCurrentYear = date('Y');
+        $dateNextYear = date('Y', strtotime('+1 year'));
+
+        $tagsSelected = $this->tagRepository->getTagsBySelected();
+
+        $confTechFetcherSynonyms = [
+            'android', //0
+            null, //1
+            null, //2
+            'css', //3
+            null, //4
+            'data', //5
+            'devops', //6
+            'dotnet', //7
+            'elixir', //8
+            null, //9
+            null, //10
+            'general', //11
+            'golang', //12
+            null, //13
+            'graphql', //14
+            null, //15
+            'ios', //16
+            null, //17
+            'javascript', //18
+            null, //19
+            null, //20
+            'php', //21
+            'python', //22
+            null, //23
+            'ruby', //24
+            'rust', //25
+            null, //26
+            'security', //27
+            'tech-comm', //28
+            'ux', //29
+        ];
+
+        $tagMatch = array_combine(TagEnum::toArray(), $confTechFetcherSynonyms);
+
+        $params = [
+        $dateCurrentYear => [],
+        $dateNextYear => [],
+    ];
+
+        foreach ($tagsSelected as $tag) {
+            /* @var Tag $tag*/
+            if (null !== $tagMatch[$tag->getName()]) {
+                array_push($params[$dateCurrentYear], [$tagMatch[$tag->getName()], $tag->getName()]);
+                array_push($params[$dateNextYear], [$tagMatch[$tag->getName()], $tag->getName()]);
+            }
+        }
+
+        return $params;
+    }
+
+    private function hydrateConference($fC)
+    {
+        $conference = new Conference();
+        $conference->setSource($fC->source);
+        $conference->setHash($fC->hash);
+        $conference->setSlug($fC->slug);
+        $conference->setName($fC->name);
+        $conference->setLocation($this->getLocation($fC));
+        $conference->setStartAt(\DateTime::createFromFormat('Y-m-d h:i:s', $fC->startDate.' 00:00:00'));
+        $conference->setEndAt($fC->endAt);
+        $conference->setSiteUrl($fC->url);
+        $conference->addTag($fC->tag);
+
+        if (isset($fC->description)) {
+            $conference->setDescription($fC->description);
+        }
+
+        if (isset($fC->cfpUrl)) {
+            $conference->setCfpUrl($fC->cfpUrl);
+        }
+
+        if (isset($fC->cfpEndDate)) {
+            $cfpEndAt = \DateTime::createFromFormat('Y-m-d h:i:s', $fC->cfpEndDate.' 00:00:00');
+            $conference->setCfpEndAt($cfpEndAt);
+        }
+
+        return $conference;
+    }
+
+    private function hash(object $fC)
```

Comment:
This method must be extracted to an isolated service, because we will need it in all Fetchers